src/style.css

```diff
@@ -1 +1,60 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bars {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, sans-serif;
```

Comment:
You've specified `Avenir, sans-serif` as the `font-family`. However, you haven't provided a fallback font in case `Avenir` doesn't load. Consider adding a generic fallback font like `serif` or `sans-serif`. [Explanation](https://www.w3schools.com/cssref/pr_font_font-family.asp)

Reply:
It is already set to sans serif.